webdriver-spec.html

```diff
@@ -6546,185 +6556,327 @@
-              the <a>current browsing context</a>’s <a>innerHeight</a>).
+        <li><p>Add <var>button</var> to the set corresponding
+            to <var>input state</var>'s <code>pressed</code>
+            property, and let <var>buttons</var> be the resulting
+            value of that property.</li>
 
-            <li><p>Let <var>x</var> be (<var>visible width</var> / 2.0).
+        <li><p><a href="perform-actions">Perform
+            implementation-specific action dispatch steps</a>
+            equivalent to pressing the button
+            numbered <var>button</var> on the pointer with
+            id <var>source id</var>, having type
+            type <var>pointerType</var> at viewport x
+            coordinate <var>x</var>, viewport y
+            coordinate <var>y</var>, with buttons <var>buttons</var>
+            depressed, in accordance with the requirements of
+            [[!UI-EVENTS]] and [[!POINTER-EVENTS]], with
+            <var>primary</var> determining whether the pointer is
+            regarded as a primary pointer and thus generates legacy
+            mouse events. Type specific properties for the pointer
+            that are not exposed through the webdriver API must be set
+            to the default value specified for hardware that doesn't
+            support that property.</li>
+
+        <!-- TODO: add some events that should be emitted? This is a
+        bit complicated in this case because e.g. pointerDown is only
+        emitted if there were no previous buttons held down --->
 
-            <li><p>Let <var>y</var> be (<var>visible height</var> / 2.0).
-          </ol>
-          <li><p>Let <var>button</var> be the result of <a>getting a property</a>
-            named "<code>button</code>".
-            <p class=issue>Do we need symbols here?
-            <p class=issue>Do we need to base events off source? E.g. if a pen?
-          <li><p>Generate <a data-lt="mouseup Event">mouseup</a> <a>Mouse Event</a> against
-              <var>element</var>.
-              <p class=issue>What about touch/pointer events?
+        <li><p>Return <a>success</a> with data null.</li>
       </ol>
 
-    <p>The <dfn>pointerDown</dfn> command:
+      <p>When required to <dfn>dispatch a pointerUp action</dfn> with
+        arguments <var>source id</var>, <var>action
+        object</var>, <var>input state</var> and <var>tick
+        duration</var> a <a>remote end</a> must run the following
+        steps:</p>
+
       <ol>
-        <li><p>If the <a>current top-level browsing context</a> is <a>no longer open</a>
-          , return <a>error</a> with <a>error code</a> <a>no such window</a>.
+        <li><p>Let <var>pointerType</var> be equal to <var>action
+              object</var>'s <code>pointerType</code> property.</li>
 
-        <li><p><a>Handle any user prompts</a> and return its value if it is an
-          <a>error</a>.
+        <li><p>Let <var>button</var> be equal to <var>action
```

Comment:
Not if you want to support holding down the left button and then the right button and then releasing in either order.